css-two-column-layout/styles.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: Karma, sans-serif;
+  letter-spacing: 2px;
+}
+
+.container {
+  margin: 5%;
```

Comment:
The `container` class typically has a `max-width` and `margin-left` and `margin-right` of `auto` 🤔